frontend/js/Pages/User/Login.vue

```diff
@@ -54,12 +54,12 @@
             </label>
           </div>
           <div class="text-sm">
-            <a
-              href="#/forgotpassword"
+            <InertiaLink
+              href="/reset"
```

Comment:
`Reset` what? Maybe `reset-password` will be better or `password/reset`? Or `forgot-password`?